crates/rolldown/src/utils/tweak_ast_for_scanning.rs

```diff
@@ -29,19 +27,6 @@ impl<'a, 'ast> PreProcessor<'a, 'ast> {
 
 impl<'ast, 'a: 'ast> VisitMut<'ast> for PreProcessor<'a, 'ast> {
   fn visit_program(&mut self, program: &mut ast::Program<'ast>) {
-    if self.has_lazy_export {
```

Comment:
This hack is used because `oxc` will parse 
```js
"literal"
```
as directive, now we use `("literal")` instead, so not needed